style/layout/_main.scss

```diff
@@ -0,0 +1,197 @@
+.main {
+    display: flex;
+    flex-direction: row;
+
+    &__heading {
+        font-family: $primary-font;
+        font-size: 5rem;
+        font-weight: 600;
+        margin: 4rem 2rem;
+    }
+
+    &__posts {
+        display: flex;
+        flex-direction: column;
+    }
+}
+
+.post {
+    display: flex;
+    flex-direction: row;
+
+    &__img {
+        display: block;
+        width: 30rem;
+        height: 20rem;
+        margin: 0 2rem 2rem 2rem;
+
+        &--1 {
+            background: url('../../img/fire1.svg');
+            background-size: cover;
+            width: 52rem;
+            height: 20rem;
+            margin: 2rem 2rem 2rem 2rem;
+        }
+
+        &--2 {
+            background: url('../../img/leaf-home.svg');
+            background-size: cover;
+            width: 52rem;
+            height: 20rem;
+            margin: 2rem 2rem 2rem 2rem;
+        }
+
+        &--3 {
+            background: url('../../img/meal.svg');
+            background-size: cover;
+            width: 52rem;
+            height: 20rem;
+            margin: 2rem 2rem 2rem 2rem;
+        }
+    }
+
+    &__textbox {
+        display: block;
+        margin: 2rem 0;
+    }
+
+    &__subheading {
+        color: $color-primary;
+        font-family: $primary-font;
+        font-size: $font-size-small;
+        text-transform: uppercase;
+        font-weight: 600;
+        margin: 2rem 0;
+    }
+
+    &__heading {
+        font-size: 3rem;
+        margin: 0;
+    }
+
+    &__paragraph {
+        margin: 2rem 0;
+        line-height: 2;
+    }
+
+    &__morebox {
+        position: relative;
+    }
+
+    &__more {
+        font-family: $primary-font;
+        color: $color-black;
+        font-size: $font-size-small;
+        font-weight: 600;
+        text-decoration: none;
+        text-transform: uppercase;
+        margin-left: 4rem;
+
+        &::before {
+            content: "+";
+            font-family: $primary-font;
+            font-size: $font-size-big;
+            color: $color-primary;
+            display: inline-block;
+            transform: rotate(-45deg);
+            position: absolute;
+            bottom: -2rem;
+            left: 0;
+        }
+    }
+}
+
+.mission {
+    margin: 2rem;
+    margin-top: 6.5rem;
+    max-width: 40%;
+    position: relative;
+
+    &__back-subheading {
+        font-family: $primary-font;
+        color: $color-white;
+        font-size: $font-size-xxl;
+        font-weight: 500;
+        text-transform: uppercase;
+        word-break: break-all;
+        line-height: 0.7;
```

Comment:
Rozmiar czcionki masz xxl a line-height: 0.7. Mam pewne wątpliwości czy do dobry pomysł, nie pamiętam wyglądu, ale możliwe, że jest lepszy sposób na osiągnięcie efektu, który próbujesz uzyskać. Przegadajmy to jeszcze na żywo.